db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/OrderByList.java

```diff
@@ -338,14 +339,21 @@ public void estimateCost(Optimizer optimizer,
             if (optimizer == null) {
                 double parallelCost = (baseCost.localCost()+baseCost.remoteCost())/baseCost.partitionCount();
                 baseCost.setLocalCost(baseCost.localCost()+parallelCost);
-                baseCost.setLocalCostPerPartition(baseCost.localCost(), baseCost.partitionCount());
-                baseCost.setRemoteCostPerPartition(baseCost.remoteCost(), baseCost.partitionCount());
+                baseCost.setLocalCostPerPartition(baseCost.localCost());
```

Comment:
Are you taking out the divide by partitionCount because this routine is doing base table estimation, and base table estimates aren't divided by number of partitions?

Reply:
@msirek The baseCost here is the total cost of a complete join path before sorting. For join result, we set the localCost and localCostPerPartition to be the same, so I take out the division.